Aquality.Selenium.Template/Aquality.Selenium.Template.SpecFlow/StepDefinitions/ContactUsPageSteps.cs

```diff
@@ -48,5 +54,12 @@ public void ThenNotificationAboutEmptyFieldsIsPresent()
             Assert.IsTrue(contactUsPage.IsEmailValidationMessagePresent,
                 "Email validation message should be displayed");
         }
+
+        [Then(@"Contact us page dump is difference")]
```

Comment:
Shouldn't it be "Contact us page dump is different"?

Reply:
done